libraries/llvm/rts.ll

```diff
@@ -1,5 +1,9 @@
 ; Run-Time System
 
+attributes #0 = { "ccc" }
```

Comment:
@phischu I added this in experiments of trying to fix the calling convention. Either we annotate it everywhere where we call into c or drop it here. What do you think?